app/main.py

```diff
@@ -1,3 +1,14 @@
-def cache(func):
-    # Write your code here
-    pass
+import typing
+
+
+def cache(func: typing.Callable) -> typing.Callable:
+    stored_values = {}
+
+    def wrapper(*args) -> typing.Any:
+        if args in stored_values.keys():
```

Comment:
`.keys()` redundant here
```suggestion
        if args in stored_values:
```